src/evaluators/interpolation/PHAL_DOFCellToSide.hpp

```diff
@@ -59,6 +59,7 @@ class DOFCellToSideBase : public PHX::EvaluatorWithBaseImpl<Traits>,
   enum LayoutType
   {
     CELL_SCALAR = 1,
+    CELL_SCALAR_SIDESET,
```

Comment:
So you are only producing an output with collapsed layout for the "P0-scalar" and P1-scalar/vector cases, is that correct?

That's fine, for now, but it's worth making a TODO action item, to propagate collapsed layout also to P0 vector/tensor and P1-tensor quantities.

Note: this can be done when the non-collapsed side layouts are completely removed.

Reply:
I'll add a TODO here just to be safe but yes, you have that right and my plan is to address this during the removal of the non-collapsed layouts.